modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout_set/look_and_feel.jsp

```diff
@@ -37,6 +37,26 @@ LayoutSet selLayoutSet = layoutsAdminDisplayContext.getSelLayoutSet();
 	<liferay-util:include page="/look_and_feel_theme_css.jsp" servletContext="<%= application %>" />
 </div>
 
+<clay:sheet-section>
+	<h3 class="sheet-subtitle"><liferay-ui:message key="theme-spritemap-client-extension" /></h3>
+
+	<clay:alert
+		displayType="info"
+		message='<%= LanguageUtil.get(request, "to-add-or-edit-the-existing-spritemap-simply-copy-paste-and-make-changes-as-needed-to-your-registered-extension") %>'
+	/>
+
+	<p>
+		<liferay-ui:message key="use-this-client-extension-to-fully-replace-the-default-spritemap-contained-in-the-theme" />
+	</p>
+
+	<div>
+		<react:component
+			module="js/layout/look_and_feel/ThemeSpritemapCETsConfiguration"
+			props="<%= layoutLookAndFeelDisplayContext.getThemeSpritemapCETConfigurationProps(Layout.class.getName(), selLayoutSet.getLayoutSetId()) %>"
```

Comment:
This shall be `LayoutSet.class.getName()`if we follow the pattern in line 65, right?